implementations/elixir/ockam/ockam_hub/lib/hub/service/token_lease_manager/storage_service.ex

```diff
@@ -4,9 +4,46 @@ defmodule Ockam.TokenLeaseManager.StorageService do
   @type reason :: any()
   @type lease_id :: String.t()
   @type options :: Keyword.t()
+  @type storage_conf :: map()
 
-  @callback save(lease) :: :ok | {:error, reason}
-  @callback get(lease_id) :: {:ok, lease} | {:error, reason}
-  @callback remove(lease_id) :: :ok | {:error, reason}
-  @callback get_all(lease) :: {:ok, [lease]} | {:error, reason}
+  @callback do_init(options) :: {:ok, pid()} | {:error, reason}
```

Comment:
I personally prefer to use `do_` prefix for internal function and not callbacks.
If that's a callback module for a gen-server, maybe we can use the `handle_` prefix instead?
Same for the cloud_service

Reply:
sure